frontend/src/ClientDataTable.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './App.css'
+
+
+const ClientDataList = () => {
+    const [data, setData] = useState([]); //
+      
+        //Fetch data using axios
+    useEffect(() => {
+        console.log("App.js useEffect called.");
+        axios.get('http://127.0.0.1:8000/clientdata/')
+        .then(response => {
+            setData(response.data);
+        })
+        .catch(error => {
```

Comment:
I'm getting `Network Error` on this line.
See screenshot below:
<img width="278" alt="Screen Shot 2024-05-28 at 11 44 09 AM" src="https://github.com/smithev95/NEFP-Capstone/assets/145937137/a5c7c2fa-6c22-43a4-8105-d76db6ed44b0">

Reply:
I think this has to do with setting `CORS`.
A resource that might be helpful: https://www.stackhawk.com/blog/react-cors-guide-what-it-is-and-how-to-enable-it/